src/BowlingController.js

```diff
@@ -0,0 +1,141 @@
+$('document').ready(function(){
+  
+  var player
+  var scoreboard
+  var frame; 
+  var rollsArray
+  var clicks
+
+  newGame();
+  rackupFrame();
+  
+  function newGame(){
+    player = new Player
+    scoreboard = new ScoreBoard
+    clicks = 0
+    unpopulateTable();
+    unpopulateRollTable();
+  }
+
+  function rackupFrame(){
+    player.newFrame();
+    frame = new Frame;
+    displayPins();
+    textTakeARoll();
+  };  
+
+  function displayPins(){
+    $('#pins-remaining').text(frame.pinsRemaining);
+  };
+
+  function textTakeARoll(){
+    $('#take-roll').text("Take a roll!");
+  }
+
+  function textNextFrame(){
+    $('#take-roll').text("Next frame!");
+  }
+
+  function textNewGame(){
+    $('#take-roll').text("New Game!");
+  }
+
+  function getRoll(){
+    return player.roll();
+  };
+
+  function isRollOneDone(){
+    return frame.rollOneDone;
+  };
+
+  function isRollTwoDone(){
+    return frame.rollTwoDone;
+  };
+
+  $('#take-roll').click(function(e){
+    e.preventDefault();
+    clicksCounter();    
+    if(!isRollOneDone()) return rollOne();
+    if(!isRollTwoDone()) return rollTwo();
+  });
+
+  function clicksCounter(){
+    clicks++;
+    if(clicks===21) textNewGame();
+    if(clicks===22) location.reload();
+  }
```

Comment:
Counting clicks is ingenious but probably not the best way to see if the game is complete. You are mixing your concerns here - the user interface and the logic should be separate, but here they have a lot of dependency.